core/css.js

```diff
@@ -360,7 +360,7 @@ Blockly.Css.CONTENT = [
     'fill: #000;',
   '}',
 
-  '.blocklyEditableText:hover>rect {',
+  '.blocklyEditableText:not(.editing):hover>rect {',
```

Comment:
Firefox, Safari include the stroke when calculating sizing info. This ensures that when we calculate the size, there's not stroke.